camus-kafka-coders/pom.xml

```diff
@@ -28,6 +28,12 @@
       <artifactId>json-smart</artifactId>
       <version>${json-smart.version}</version>
     </dependency>
+    <dependency>
+      <groupId>junit</groupId>
```

Comment:
yes, after switching to the new version of cdh, camus-kafka-coder started to fail due to not being able to find junit. I guess the old version pulled in junit.